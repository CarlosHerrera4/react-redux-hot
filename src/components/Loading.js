import React from 'react';
import LoadingImg from '../assets/loading.gif';

const Loading = () => <img src={LoadingImg} alt='loading' />;

export default Loading;
