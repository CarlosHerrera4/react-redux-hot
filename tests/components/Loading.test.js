import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../../src/components/Loading';
// import Image from '../../src/assets/loading.gif';

test('Loading renders as expected', () => {
  const component = renderer.create(
    <Loading />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
