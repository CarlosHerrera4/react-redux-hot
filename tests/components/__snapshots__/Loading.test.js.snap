// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders as expected 1`] = `
<img
  alt="loading"
  src="test-file-stub"
/>
`;
